Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,54 @@
 // 1.	Slide the header up when it is clicked on.
+$('header').on('click', function() {
+	$('header').slideUp();
+});
 
 // 2.	Remove the purplebox class
+$('article').removeClass('purplebox');
 
 // 3.	Hide all of the div’s in #row4
+$('#row4').hide('div');
```

Comment:
So close! Just a little off. The `.hide()` method accepts some options that relate to easing and completion(http://api.jquery.com/hide/), but not _what to hide_ — that is done with the selection. So effectively you hid the entire #row4, and then passed the .hide() method an unrecognized argument. You want to just select what it is you want to hide and then use `.hide()`, like:
```
$('#row4 div').hide();
```